spiro.c

```diff
@@ -1231,6 +1231,12 @@ spiro_to_bpath(const spiro_seg *s, int n, bezctx *bc)
     spiro_to_bpath0(NULL, s, dm, SPIRO_RETRO_VER1, n, bc);
 }
 
+const char *
```

Comment:
You should get rid of this function and just add the version to the header. The fact that you can't detect what version of libspiro you are using based on inspection of headers is bad, and goes against what all modern packages do.

Reply:
I'm thinking further than just FontForge use :-)

Reply:
No, there is literally no use case for this function if your headers properly define what the version is, like any good package should.

Reply:
I wasn't even thinking about packaging when I added the function.
...and scripting is a good example.

Would you prefer to run a script that goes weird, or dies, or gives bad results, or would you instead prefer to run a script that tests for a certain limit and then continues, or else says "halt, I cannot run - please update your program, or library."?